code/modules/atmospherics/environmental/LINDA_turf_tile.dm

```diff
@@ -407,6 +411,8 @@
 	var/display_id = 0
 	///Wrapping loop of the index colors
 	var/static/wrapping_id = 0
+	///If there are turfs in the group with gasmixtures with reactions occuring.
+	var/has_reactions = NONE
```

Comment:
call this turf_reactions, and describe it as all turf reaction flags we've received. what you've got now is a bit unclear